cargo-dist/src/cli.rs

```diff
@@ -84,6 +85,15 @@ pub enum Commands {
     Manifest(ManifestArgs),
 }
 
+#[derive(Args)]
+pub struct ReleaseNotesArgs {
+    /// Get release notes for a specific version.
+    ///
+    /// Otherwise the app's current version will be used.
+    #[clap(long)]
+    pub version: Version,
```

Comment:
i wonder if we will get people who don't want to use semver.... (this is not a change request, just random thought)

Reply:
within cargo itself you have no option but to use Semver